.bundle/config

```diff
@@ -0,0 +1,3 @@
+---
+BUNDLE_PATH: "./vendor"
+BUNDLE_DISABLE_SHARED_GEMS: '1'
```

Comment:
Could you explain why you chose to start vendoring the gems? Is this necessary for your setup?